test/functional/tests/initialize/test_startup_init_config.py

```diff
@@ -441,29 +444,29 @@ def test_failover_config_startup():
         power_control.power_cycle()
 
     with TestRun.step("Verify if all the devices are initialized properly"):
-        core_pool_list = get_cas_devices_dict()["core_pool"]
-        caches_list = get_cas_devices_dict()["caches"].values()
-        cores_list = get_cas_devices_dict()["cores"].values()
+        core_pool_list = casadm_parser.get_cas_devices_dict()["core_pool"]
+        caches_list = casadm_parser.get_cas_devices_dict()["caches"].values()
+        cores_list = casadm_parser.get_cas_devices_dict()["cores"].values()
 
         if len(core_pool_list) != 0:
             TestRun.error(f"No cores expected in core pool. Got {core_pool_list}")
         else:
             TestRun.LOGGER.info("Core pool is ok")
 
-        expected_caches_paths = set([active_cache_path, standby_cache_path])
+        expected_caches_paths = {active_cache_path, standby_cache_path}
```

Comment:
Not sure if active_cache_path is correct naming for cache_disk_path

Reply:
I assume those names gives us more info what are there caches than "first_cache" and "second_cache" names